crates/astria-api/src/sequencer/v1alpha1/transaction/mod.rs

```diff
@@ -0,0 +1,263 @@
+use ed25519_consensus::{
+    Signature,
+    SigningKey,
+    VerificationKey,
+};
+use prost::Message as _;
+
+use super::raw;
+
+pub mod action;
+pub use action::Action;
```

Comment:
I wasn't sure about whether or not to re-export the individual actions here as well.